src/sentry/static/sentry/app/components/events/interfaces/generic.tsx

```diff
@@ -1,14 +1,13 @@
 import PropTypes from 'prop-types';
 import React, {Component} from 'react';
 
-import SentryTypes from 'app/sentryTypes';
 import ButtonBar from 'app/components/buttonBar';
 import Button from 'app/components/button';
 import EventDataSection from 'app/components/events/eventDataSection';
 import KeyValueList from 'app/components/events/interfaces/keyValueList/keyValueList';
 import {t} from 'app/locale';
 
-function getView(view, data) {
+function getView(view: View, data: State['data']) {
```

Comment:
it could also be 
```suggestion
function getView(view:  State['view'], data: State['data']) {
```

but I think it's fine :)

Reply:
I think it's fine considering View is specifically typed, I'm going to merge it in for now 😄